src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -561,6 +561,19 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    * @}
    */
 
+  /**
+   * Callback signature used for an unsuccessful operation on an
+   * element in a batch.
+   *
+   * This must be called for errors that are specific to a particular
+   * reference in a batch. Exceptions can be thrown to indicate a
+   * whole-batch error.
+   *
+   * The appropriate error code should be used for these errors. See
+   * @fqref{BatchElementError.ErrorCode} "ErrorCode".
+   */
+  using BatchElementErrorCallback = std::function<void(std::size_t, const BatchElementError&)>;
```

Comment:
[question] Is there any merit in a common declaration for these? Do we ever imagine changing the callback signature between `Manager`/`ManagerInterface`?

Reply:
Cool, thanks, what do you think about the duplication of the other callbacks in `Manager` and `ManagerInterface` too?

Reply:
It's a tricky one. From a technical perspective, if they were to share an alias, which header would it go in?  From a design perspective, are they only coincidentally the same signature and should really be seen as independent?

Reply:
👍 lets leave it as-is for now. Thanks.